client/gutenberg/extensions/map/settings.js

```diff
@@ -17,6 +17,7 @@ export const settings = {
 	),
 	category: 'jetpack',
 	keywords: [ __( 'map' ), __( 'location' ) ],
+	description: __( 'Add an interactive map showing one or more locations.' ),
```

Comment:
Just out of curiosity, why aren't these in editor.js?

Reply:
It has to do with what this block is doing on the view side. The view-side renderer needs access to a few global variables (`name` and `attributes`), which are normally defined in `editor.js`. To avoid duplicating the definitions I began keeping them in a file dedicated to static block data - `settings.js` - and importing the file into `editor.js` and `view.js`. When this proved successful I ended up going further and put nearly all static data points in this file to make it easy to find things.

Reply:
Oh cool. Could you do a bit of a writeup of this if you think it would be a useful pattern to follow in block development?

Reply:
Wrote about this (and other aspects of the block) in `p89M8K-8j-p2`. After the block launches, I'll revise the post and possibly post somewhere public.

Reply:
Perfect